docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,6 +156,27 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
+## Setting server icon URI
```

Comment:
@m-michalski2 
Thank you very much for adding "getting" in line number: 22.
Please update line number: 159 also.
To maintain consistency please use title case for this heading.

Setting and

Reply:
done